doc/source/whatsnew/v0.24.0.txt

```diff
@@ -18,6 +18,8 @@ Other Enhancements
 - :func:`Series.mode` and :func:`DataFrame.mode` now support the ``dropna`` parameter which can be used to specify whether NaN/NaT values should be considered (:issue:`17534`)
 - :func:`to_csv` now supports ``compression`` keyword when a file handle is passed. (:issue:`21227`)
 - :meth:`Index.droplevel` is now implemented also for flat indexes, for compatibility with MultiIndex (:issue:`21115`)
+- :func:`to_gbq` and :func:`read_gbq` signature and documentation updated to
```

Comment:
can you add here a link to the pandas-gbq docs whatsnew for 0.5.0?

Reply:
Added link to https://pandas-gbq.readthedocs.io/en/latest/changelog.html#changelog-0-5-0